networkit/cpp/centrality/test/DynBetweennessGTest.cpp

```diff
@@ -134,5 +137,80 @@ TEST_F(DynBetweennessGTest, runApproxBetweenness) {
 	DEBUG("Number of samples: ", bc1.numberOfSamples());
 }
 
+TEST_F(DynBetweennessGTest, runDynVsStaticCaseInsertDirected){
+	srandom(0);
+	Aux::Random::setSeed(0, false);
+
+	auto genEdgeInsert = [](const Graph& g){
+		int n = g.numberOfNodes();
+		while(true){
+			int u = random() % n, v = random() % n;
+			if(u == v or g.hasEdge(u, v))continue;
+			return std::make_pair(u, v);
+		}
+		//return {-1, -1};
+	};
+
+	for(int n = 2; n <= 25; n++)
+		for(int t = 0; t < 100; t++){
+			auto g = ErdosRenyiGenerator(n, 0.3, true).generate();
+
+			if(g.numberOfEdges() == g.numberOfNodes() * (g.numberOfNodes() - 1))continue;
+
+			auto e = genEdgeInsert(g);
+			auto ibet = DynBetweenness(g);
+
+			ibet.run();
+
+			auto ne = GraphEvent(GraphEvent::EDGE_ADDITION, e.first, e.second);
+			g.addEdge(e.first, e.second);
+			ibet.update(ne);
+
+			assert(g.hasEdge(e.first, e.second));
+			auto brandes = Betweenness(g);
+			brandes.run();
+			g.forNodes([&](node & v){
+				EXPECT_NEAR(brandes.score(v), ibet.score(v), 1e-8);
+			});
+		}
+}
+
+TEST_F(DynBetweennessGTest, runDynVsStaticCaseInsertUndirected){
+	//srandom(0);
+	//Aux::Random::setSeed(0, false);
```

Comment:
```suggestion
	Aux::Random::setSeed(0, false);
```
The test should be be reproducible.